### DISCUSSION TITLE
Searching in TinyDB

### DISCUSSION THREAD

**Discussion by wtopazi** [msiemens/tinydb#489] [Q&A]
Is there a way to create individual, searchable entities in the same field? For example, if I had a database of users who had multiple pets, but I wanted to search for all those who have cats.

I hope that makes sense, but let me know if any clarification is needed.

Thanks!

**Comment by msiemens** [comment]
Hey @wtopazi, do I understand correctly that you want something like this: `{"name": "msiemens", "pets": ["cat", "dog"]}`? You can use a regular list as in the example and then query like this:

```python
>>> db.search(where('pets').any('cat'))
[{'name': 'msiemens', 'pets': ['cat', 'dog']}]

>>> db.search(where('pets').any('bird'))
[]
```

Does that solve your problem?
